Documentation/bpf.rst

```diff
@@ -1119,13 +1119,13 @@ following commands can be used:
 
 ::
 
-    $ git clone https://git.llvm.org/git/llvm.git
-    $ cd llvm/tools
-    $ git clone --depth 1 https://git.llvm.org/git/clang.git
-    $ cd ..; mkdir build; cd build
-    $ cmake .. -DLLVM_TARGETS_TO_BUILD="BPF;X86" -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_BUILD_RUNTIME=OFF
+    $ git clone https://github.com/llvm/llvm-project.git
+    $ cd llvm-project
+    $ mkdir build
+    $ cd build
+    $ cmake -DLLVM_ENABLE_PROJECTS=clang -DLLVM_TARGETS_TO_BUILD="BPF;X86" -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_BUILD_RUNTIME=OFF  -G "Unix Makefiles" ../llvm
     $ make -j $(getconf _NPROCESSORS_ONLN)
-
+    
```

Comment:
Could you also remove this white space change if it is not needed?

Reply:
Sure, just pushed the fix